hack/cve/trivy.yaml

```diff
@@ -0,0 +1,20 @@
+# Ref: https://aquasecurity.github.io/trivy/v0.33/docs/references/customization/config-file
```

Comment:
nit: I think we can keep config files like this at the top level directory (see others like `.codecov.yaml` or `.golangci.yaml`). Just make sure to prefix with a `.`.

Reply:
I'll move the config to project root.